solutions/swb-reference/src/SWBStack.ts

```diff
@@ -807,13 +792,13 @@ export class SWBStack extends Stack {
     return bucket;
   }
 
-  private _createSecureS3Bucket(s3BucketId: string, s3OutputId: string, mainAcctEncryptionKey: Key): Bucket {
+  private _createSecureS3Bucket(s3BucketId: string, s3OutputId: string, encryptionKey: Key): Bucket {
```

Comment:
@janeyuaws there is construct defined for SecureS3Bucket. Please check if you can use it [here](https://github.com/aws-solutions/solution-spark-on-aws/blob/janeyu/key-rotation/workbench-core/infrastructure/src/workbenchSecureS3Bucket.ts)

Reply:
acknowledged but out of scope for this PR